config.go

```diff
@@ -10,6 +10,7 @@ import (
 // Configuration is a minimal set of parameters for Sentry integration.
 type Configuration struct {
 	Tags              map[string]string
+	NameKey           string
```

Comment:
it would be nice to left the comment what is for and rename it as LoggerNameKey

Reply:
Isn't it a bit excessive to add the prefix?
There is `Level` field, which could be `LoggerLevel`, etc.
I don't mind to rename it, just wanted to stay consistent with the other configuration fields.

Reply:
@TheZeroSlave Renamed the field. Added comments to all the configuration fields.

Reply:
`LoggerName` is kind of specific to Sentry - even in Sentry it is called exactly that, so I agree with the name change.